SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -449,11 +460,6 @@ void CIncEulerSolver::SetNondimensionalization(CConfig *config, unsigned short i
   const su2double MassDiffusivityND = config->GetDiffusivity_Constant() / (Velocity_Ref * Length_Ref);
   config->SetDiffusivity_ConstantND(MassDiffusivityND);
 
-
-  /*--- Delete the original (dimensional) FluidModel object. No fluid is used for inscompressible cases. ---*/
-
-  delete auxFluidModel;
```

Comment:
Memory leak, where is this deleted now?

Reply:
Thanks, I will delete it immediately after the last use of auxFluidModel